### PR TITLE
exporters/datadog: remove trace support for windows for now

### PR DESCRIPTION
**Pull request by ericmustin**
This PR removes windows support for trace export for the datadog exporter for windows as it relies on packages which use cgo/gcc, which are causing the `windows-test` build step to fail https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/1228. 

It should fix the failing build on master, cc @tigrannajaryan @mx-psi @james-bebbington

### CODE REVIEW

**Review by tigrannajaryan — APPROVED** [review]
LGTM, provided that the build passes.

**Review by mx-psi — APPROVED** [review]
LGTM, the tests passed

**Comment by tigrannajaryan** [comment]
@ericmustin  Mergify is stuck and I can't merge the PR. Can you touch the commit (just edit the commit message) so that the checks are re-trigerred? I disabled Mergify, it will not run anymore, but I can't stop it if it is already running.

**Comment by mx-psi** [comment]
I made an empty commit to retrigger the CI, looks like Mergify is gone now :)

**Comment by tigrannajaryan** [comment]
@mx-psi thanks. Now EasyCLA is stuck for some reason. Can you try bumping with another commit again or if it does not help we may need to close and reopen the PR.

**Comment by bogdandrutu** [comment]
@tigrannajaryan It is not going to help, it is broken everywhere. Another way to retrigger things is to close and reopen the PR

**Comment by ericmustin** [comment]
@bogdandrutu @tigrannajaryan would you like me to close / re-open this PR?

**Comment by tigrannajaryan** [comment]
@ericmustin @mx-psi thank you for the quick fix. Let's see if the master build now passes.

**Comment by tigrannajaryan** [comment]
We are good, the build passed.

**Pull request merged**
